src/App.tsx

```diff
@@ -2,21 +2,46 @@ import * as React from 'react';
 import './App.css';
 import { JamInput } from './JamInput';
 import { JamText } from './JamText';
+import { Flex } from 'react-flex';
+import 'react-flex/index.css';
 
-const logo = require('./logo.svg');
+const logo = require('./img/logo.svg');
+const atom = require('./img/atom.png');
 
 class App extends React.Component {
     render() {
         return (
             <div className="App">
-                <div className="App-header">
-                    <img src={logo} className="App-logo" alt="logo"/>
-                    <h2>Welcome to Slichter's Jams</h2>
-                </div>
-                <div>
-                    <JamInput defaultMessage="test"/>
-                    <JamText/>
+                <div id="header-bar">
+                    <Flex justifyContent="space-between">
+                        <div id="logo">
+                            <img src={logo}/>
+                        </div>
+                        <div id="atom">
+                            <img src={atom}/>
+                        </div>
+                    </Flex>
                 </div>
+
+                <div className="vert-spacer"/>
+
+                <Flex column={true} alignItems="center">
```

Comment:
What's going on here: `column={true}`?

Reply:
https://www.npmjs.com/package/react-flex

Reply:
👌

Reply:
It's an npm library that @tleyda had added. But we are considering ripping it out and just using normal css...